sbws/__init__.py

```diff
@@ -1 +1,20 @@
 __version__ = '1.0.3-dev0'
+
+
+import threading
+
+from . import globals  # noqa
+
+
```

Comment:
Hmm, I don't quite understand what's going on here and the commit does not explain what this is used for?
Perhaps add some more docs to explain this addition to the init file? What are we trying to achieve?